packages/main/src/Input.js

```diff
@@ -526,6 +576,40 @@ class Input extends UI5Element {
 		}
 	}
 
+	toggle(isToggled) {
+		if (isToggled) {
+			this.openPopover();
+		}
```

Comment:
You can be sure that in the "else" statement "isToggled" would be false, so you can skip checking it.
And, use "isOpen", instead if "this._isPopoverOpen".

And, better 
```js
if (isToggled) {
		this.openPopover();
} else {
	    this.closePopover(); // if you want you can make the check for isOpen in the "closePopover"
}
```